communication guidelines.md

```diff
@@ -0,0 +1,77 @@
+## Purpose:
+[GitHub Actions](https://docs.github.com/en/actions/learn-github-actions) gives you the flexibility to build an automated software development lifecycle workflow. With [GitHub Actions for Azure](https://docs.microsoft.com/en-gb/azure/developer/github/) you can create workflows that you can set up in your repository to build, test, package, release and deploy to Azure
+This document is to set communication guidelines, processes for making GitHub Actions ready for the external community. We want to set a robust framework for communication, and governance for anything related to Azure Actions, for enabling both internal and external users to be always updated on any change in this area.
+
+## Background information needed:
+1. Microsoft open source program office’s protocols and definitions.
+2. We would use GitHub aliases for all the communication
+3. All the communication (except for a few cases) we would want all communication to be public Do we ever need private communication? Is there anything proprietary? 
+4. In case of communication that involves strategic decisions which partner team would not want to go public earlier official announcement, have all the communication and key decisions in the form of GH Discussions/Issues inside the private repo. Once right time arises, the repo would be made public
+
+## What should be Open-Sourced:
```

Comment:
Should we say "What should be communicated" in accordance with this document as it is on communication guidelines?

Reply:
Updated